CHANGELOG.md

```diff
@@ -4,7 +4,15 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 ## Unreleased
 
+### Added
+
+- Added a new in-game player model selector
+- Added new convars that can change the way playermodels are selected (these can be found in the gamemode menu)
+- Added a new ConVar `ttt2_use_custom_models` (def: 0) to enable the custom player model selector
```

Comment:
i would maybe make the convar changes subitems of the playermodel(?) selector change. But i'll leave that up to you